pkgs/tools/networking/opensnitch/daemon.nix

```diff
@@ -15,13 +15,13 @@
 
 buildGoModule rec {
   pname = "opensnitch";
-  version = "1.5.0";
+  version = "1.5.1";
 
   src = fetchFromGitHub {
     owner = "evilsocket";
     repo = "opensnitch";
-    rev = "v${version}";
-    sha256 = "sha256-vtD82v0VlaJtCICXduD3IxJ0xjlBuzGKLWLoCiwPX2I=";
+    rev = "refs/tags/v${version}";
```

Comment:
Is this necessary? I had a look at the repo and I don't see a branch with the same name as the tag (v1.5.1)
Maybe it was there but it's gone now

Reply:
perhaps this is a new convention? If so it would be good to mention that at https://nixos.org/manual/nixpkgs/unstable/#fetchfromgithub though.

Reply:
I saw it the other day in a package when there was a tag + branch with the same name but IDK if it's worth doing all the time.
At that point we could put a tag attribute on `fetchFromGitHub` and have it fill that `refs/tags/` part in for us

Reply:
It's coming from the updateScript here: https://github.com/NixOS/nixpkgs/commit/7f5915b04801f383a3504a011601e2fb09dfedf8

Reply:
No, its not necessary all the time, only when both a branch and a tag named the same exist.